### ISSUE TITLE
Add `profile_ranking_avg` and `reduction_targets_avg` columns at company level in emission and sector profile indictors repectively

### ISSUE THREAD

**Issue by kalashsinghal** [2DegreesInvesting/tiltIndicator#726]
Relates to [comment](https://github.com/2DegreesInvesting/tiltIndicatorAfter/pull/152#discussion_r1486251040)

**Comment by maurolepore** [comment]
@kalashsinghal how about doing this in tiltIndicatorAfter? 

Computing the `mean()` of a column seems like a much better fit for that package. It's similar to jittering co2 values -- which we also do in tiltIndicatorAfter.

In the comment you point me to I see no mention of the need to add this feature  in tiltIndicator. The only indication I see is Anne's use of "emission_profile" instead of "profile_emissions" and of "sector_profile" instead of "profile_sector" -- but that may just be a confusion?

cc @AnneSchoenauer?

**Comment by kalashsinghal** [comment]
@maurolepore You are right that this task can be done in `tiltIndicatorAfter`. I thought of doing it in `tiltIndicatorAfter` but didn't see an approach that could have just work just fine (rather I only saw a complex solution). I will shift this work to tiltIndicatorAfter.
